imap_processing/tests/ena_maps/data/test_ultra_l2_map_config.json

```
@@ -0,0 +1,13 @@
+{
+    "sky_tiling_type": "HEALPIX",
+    "spice_reference_frame": "ECLIPJ2000",
+    "values_to_push_projection": [
```

I think you use "project" everywhere above.
```suggestion
    "values_to_push_project": [
```

---

Ah - I think I missed the need to `git rm` this. It's no longer used!